addons/sms/static/src/components/phone_field/phone_field.xml

```diff
@@ -4,7 +4,7 @@
     <t t-inherit="web.PhoneField" t-inherit-mode="extension">
         <xpath expr="//*[hasclass('o_phone_link')]" position="after">
             <t t-if="props.value.length > 0">
-                <SendSMSButton record="props.record" value="props.value"/>
+                <SendSMSButton t-props="props" />
```

Comment:
all props, or just what is necessary (resId, resModel, name, value)

Reply:
I keep all props, cause as soon as we need resId and resModel, this is ORM linked so having them seperated doesn't bring any advantage